tabular/src/autogluon/tabular/models/tabular_nn/torch/tabular_nn_torch.py

```diff
@@ -562,9 +562,34 @@ def _get_default_resources(self):
         num_gpus = 0
         return num_cpus, num_gpus
 
+    def save(self, path: str = None, verbose=True) -> str:
```

Comment:
add brief docstring describing why custom `save` method is necessary

Reply:
This will override superclass docstring; I believe the inline/code explain it sufficiently.